chemicaltoolbox/openduck/openduck_run_smd.xml

```diff
@@ -0,0 +1,80 @@
+<tool id="openduck_run_smd" name="Run OpenDUck" version="@VERSION@">
+    <description>steered molecular dynamics runs</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        duck_smd_runs
+            -i $input
+            -p $pickle
+            -n $num
+            -l $mdlen
+            -d $start_dist
+            -v $vel
+            #if $OPENDUCK_GPU_PARAM:
```

Comment:
this variable does not exist ... if you what to consider an environment variable you need to include something like the GALAXY_SLOTS ... eg. `\${OPENDUCK_GPU_PARAM:-1}`

I would even say, the admin need to set `export OPENDUCK_GPU_PARAM="--gpu-id foo" `. So you can save here the if.

Reply:
Ah okay, makes sense. Just to be clear, if `--gpu-id` is not set the tool is run on CPU only. So in this case the admin would do `export OPENDUCK_GPU_PARAM=""`.

Reply:
yes, or does not set this ENV at all. I guess `\${OPENDUCK_GPU_PARAM:-''}` should work in this case.